components/VercelChat/chat.tsx

```diff
@@ -81,7 +81,7 @@ export function Chat({ id, reportId }: ChatProps) {
         </Messages>
       )}
 
-      <div className="flex flex-col gap-4 w-full">
+      <div className="flex flex-col gap-4 w-full border shadow-lg rounded-2xl">
```

Comment:
you've added shadow to BOTH
- components/VercelChat/chat.tsx
- components/VercelChat/ChatInput.tsx
Are both necessary? Is only one working? What's the difference between these 2 component stylings?